### PR TITLE
[fr] 3.next sync

### PR DESCRIPTION
**Pull request by HavokInspiration**
I removed a duplicate item in the original version.

@cake17 @antograssiot I did not translate "match" / "matching" when translating the Router parts : I did not find a suitable word. So I "frenched it up" instead. I'm open to suggestions.

Follows #4585 & #4592.

### CODE REVIEW

**Comment by HavokInspiration** [comment]
Updated with the sync of #4591 and a fix in the English version (please correct me if the sentence was correct before I changed it).

File: en/appendices/3-4-migration-guide.rst [hunk 1]
```diff
@@ -105,9 +105,6 @@ Other Deprecations
 * ``Router::parse()`` is deprecated. ``Router::parseRequest()`` should be used
   instead as it accepts a request and gives more control/flexibility in handling
   incoming requests.
-* ``Route::parse()`` is deprecated. ``Route::parseRequest()`` should be used
```
Comment (antograssiot) [inline en/appendices/3-4-migration-guide.rst]: Why would you remove this ? 
`Router::parse()` and `Route::parse ()` are not the same and are both deprecated.

File: fr/appendices/3-4-migration-guide.rst [hunk 2]
```diff
@@ -108,6 +108,9 @@ Autres dépréciations
 * L'option ``fieldList`` pour ``Cake\ORM\Table::newEntity()`` et
   ``patchEntity()`` a été renommée en ``fields`` pour être plus cohérent avec
   les autres parties de l'ORM.
+* ``Router::parse()`` est dépréciée. ``Router::parseRequest()`` est maintenant
+  la méthode recommandée car elle accepte une request en argument et donne plus
+  de contrôles et de flexibilité dans la manipulation des requêtes entrantes.
```
Comment (antograssiot) [inline fr/appendices/3-4-migration-guide.rst]: Contrôle au singulier me choquerait un poil moins

File: fr/appendices/3-4-migration-guide.rst [hunk 3]
```diff
@@ -338,6 +342,17 @@ Routing
 
 * ``RouteBuilder::prefix()`` accepte maintenant un tableau de paramètres par
   défaut à ajouter à chaque route "connectée".
+* Les routes peuvent maintenant être "matché" sur des hosts spécifiques à
+  l'aide de l'option ``_host``.
+
+HtmlHelper
+==========
+
+* ``HtmlHelper::scriptBlock()`` n'engloge plus le Javascript dans un tag
```
Comment (antograssiot) [inline fr/appendices/3-4-migration-guide.rst]: englobe ?

File: fr/development/routing.rst [hunk 4]
```diff
@@ -657,6 +657,53 @@ et une action ``showItems()``, la route générée sera
...
+
+Vous pouvez "matcher" plusieurs méthodes HTTP en fournissant un tableau.
+Puisque que l'option ``_method`` est une clé de routage, elle est utilisée à la
+fois dans le parsing des URL et la génération des URL.
+
+Matching de Noms de Domaine Spécifiques
+---------------------------------------
+
+Les routes peuvent utiliser l'option ``_host`` pour "matcher" des noms de
+domaines spécifiques. Vous pouvez utiliser le passe-partout wildcard ``*.``
```
Comment (antograssiot) [inline fr/development/routing.rst]: J'aurai juste mis `la wildcard '*'`

File: en/appendices/3-4-migration-guide.rst (hunk 1 above)
Comment (HavokInspiration) [inline en/appendices/3-4-migration-guide.rst]: Oh, not the same base class. The sentences looked similar, I thought it was a duplication. I'll put it back.

File: fr/development/routing.rst (hunk 4 above)
Comment (HavokInspiration) [inline fr/development/routing.rst]: Effectivement, j'avais testé avec "passe-partout" pour voir comment ça lisait mais j'ai oublié de le retirer quand j'ai vu que ça faisait moche.

**Comment by HavokInspiration** [comment]
@antograssiot I fixed the PR based on your review

File: fr/appendices/3-4-migration-guide.rst [hunk 5]
```diff
@@ -157,7 +163,8 @@ Cake\ORM\EagerLoadable
     * ``config()``
     * setter part of ``canBeJoined()`` (devenue ``setCanBeJoined()``)
 Cake\ORM\EagerLoader
-    * ``matching()``
+    * ``matching()`` (``getMatching()`` devra être appelé après ``setMatching()``
```
Comment (antograssiot) [inline fr/appendices/3-4-migration-guide.rst]: appelée ?
Désolé de ne pas l'avoir vu à la première lecture. Je pusherai un commit demain si tu veux.

**Comment by antograssiot** [comment]
Looks good appart from my last comment

**Comment by HavokInspiration** [comment]
Fixed ;)

**Pull request merged**